deploy/chart/catalog_resources/rh-operators/elasticsearches.crd.yaml

```diff
@@ -17,57 +17,71 @@ spec:
         spec:
           description: Specification of the desired behavior of the Elasticsearch cluster
           properties:
+            managementState:
```

Comment:
A general question about this file, if we have default values defined and this CRD is created, does that mean a CR is created as well with these defaults? If so, I think we want to avoid that as the CLO will be creating the necessary `elasticsearch` CRs.

Reply:
I had the same question but I don't know if this means "default if parent is defined" or "default when the parent is defined and the child is not".  I was going to test it to see. I don't even know if validation is enabled

Reply:
defaults not supported by validation logic